configure.ac

```diff
@@ -86,16 +91,28 @@ PKG_CHECK_MODULES([HARFBUZZ], [harfbuzz >= 1.2.3], [
 libpng=false
 AS_IF([test "x$enable_test" = xyes || test "x$enable_compare" = xyes], [
     PKG_CHECK_MODULES([LIBPNG], [libpng >= 1.2.0], [
+        # Only used for test programs, must not be used for distribution
         CFLAGS="$CFLAGS $LIBPNG_CFLAGS"
         AC_DEFINE(CONFIG_LIBPNG, 1, [found libpng via pkg-config])
         libpng=true
     ])
 ])
 
+AS_IF([test "x$enable_unibreak" != xno], [
+    PKG_CHECK_MODULES([LIBUNIBREAK], [libunibreak >= 1.1], [
+        CFLAGS="$CFLAGS $LIBUNIBREAK_CFLAGS"
+        LIBS="$LIBS $LIBUNIBREAK_LIBS"
+        AC_DEFINE(CONFIG_UNIBREAK, 1, [found libunibreak via pkg-config])
+    ], [
+        : dnl# Don't fail if unibreak is not present
```

Comment:
Not necessarily a concern for this PR as this also applies to font backends, but I think that if the user explicitly says `--enable-foobar` and foobar isn’t found, we should fail.

Reply:
Added a commit dealing with this for all font-providers + libunibreak.